### PR TITLE
Remove the spray code from lenthall

### PR DESCRIPTION
**Pull request by geoffjentry**
There'll be a followup PR in cromwell to upgrade lenthall and move the spray code over there.

Phase 1 of the 2.12ification of our stack

### CODE REVIEW

File: src/main/scala/lenthall/spray/WrappedRoute.scala [hunk 1]
```diff
@@ -1,31 +0,0 @@
-package lenthall.spray
-
-import spray.routing._
-import spray.routing.directives.PathDirectives
-
-object WrappedRoute {
```
Comment (kshakir) [inline src/main/scala/lenthall/spray/WrappedRoute.scala]: Make sure the code is copied cromwell _and_ [thurloe](https://github.com/broadinstitute/thurloe/blob/0089c2646ee79a10f77a2ba1aad592d02220bf84/src/main/scala/thurloe/service/ThurloeServiceActor.scala#L7) too!

Comment (geoffjentry) [inline src/main/scala/lenthall/spray/WrappedRoute.scala]: Nope, I"m one step ahead of you :) That's not going to happen as per a discussion w/ the TLs overseeing thurloe.

File: build.sbt [hunk 2]
```diff
@@ -43,13 +41,8 @@ libraryDependencies ++= Seq(
     exclude("org.typelevel", "cats-laws_2.11")
     exclude("org.typelevel", "cats-kernel-laws_2.11"),
   "ch.qos.logback" % "logback-classic" % "1.2.1" % Provided,
-  "org.webjars" % "swagger-ui" % "2.2.2" % Provided,
-  "io.spray" %% "spray-routing" % sprayV % Provided,
-  "io.spray" %% "spray-http" % sprayV % Provided,
-  "io.spray" %% "spray-can" % sprayV % Provided,
   "com.typesafe.akka" %% "akka-actor" % akkaV % Provided,
-  //---------- Test libraries -------------------//
```
Comment (cjllanwarne) [inline build.sbt]: This comment seems potentially valuable still?

Comment (geoffjentry) [inline build.sbt]: You can't figure out that something called "testkit" with "Test" at the end is a test library?

Comment (geoffjentry) [inline build.sbt]: Note that we don't use this pattern in cromwell, i'm not sure why it is being done here.

Comment (kshakir) [inline build.sbt]: Cromwell is just ahead of the curve of everyone else. Cromwell used to have [the same](https://github.com/broadinstitute/cromwell/blob/0.18/build.sbt#L74). Lenthall (above),  [wdltool](https://github.com/broadinstitute/wdltool/blob/0.9/build.sbt#L38), [wdl4s](https://github.com/broadinstitute/wdl4s/blob/0.10/build.sbt#L51), and [centaur](https://github.com/broadinstitute/centaur/blob/c377c658ec47cf3c0acf7966d3f56eaa27362d2c/build.sbt#L27) all use this "old" style.

**Comment by kshakir** [comment]
Spray is dead. 💀  Long live (eventually) akka-http. 👍

File: build.sbt (hunk 2 above)
Comment (geoffjentry) [inline build.sbt]: I thought I'd seen it before but went to cromwell for the source of truth when I removed it. I like moving ahead of the curve, as evidenced by removing spray code ;)

**Comment by geoffjentry** [comment]
Closing this one, will have a new one w/ full 2.12 support & cross-version publishing

**Pull request closed without merging**